pytest_selenium/pytest_selenium.py

```diff
@@ -110,11 +110,6 @@ def pytest_runtest_makereport(__multicall__, item, call):
     extra_summary = []
     report = __multicall__.execute()
     extra = getattr(report, 'extra', [])
-    try:
-        report.public = item.keywords['privacy'].args[0] == 'public'
```

Comment:
I see `report.public` being removed, but not replaced with anything. Was it not actually being used for anything before?